sys/net/network_layer/fib/fib.c

```diff
@@ -531,6 +584,20 @@ void fib_init(void)
         prefix_rp[i] = NULL;
     }
 
+    for (size_t i = 0; i < FIB_MAX_FIB_SR; ++i) {
```

Comment:
Just a matter of taste: you could use `memset(fib_sr_table, 0, sizeof(fib_sr_table[0]) * FIB_MAX_FIB_SR);` here and respectively for the following for loops to initialize the structs. IMO this is much more readable.